config/config.go

```diff
@@ -513,6 +524,18 @@ func NewSub(validates ...Validate) (Config, error) {
 	return cfg, nil
 }
 
+// SetExternChainID sets the extern chain ID
+func SetExternChainID(id uint32) {
+	chainID.Do(func() {
+		_externChainID = id
+	})
+}
+
+// ExternChainID returns the extern chain ID
+func ExternChainID() uint32 {
```

Comment:
I can see why we have this function for now. But ideally, we should have a Signer and a Verifier which were created with cfg.EVMNetworkID. Global function is not a good choice, but we can leave it to future.